### PR TITLE
Adding SIMD for UTF-16 parsing.

### PR DESCRIPTION
**Pull request by CarlVerret**
In response to issue #59

### CODE REVIEW

File: Benchmark/Benchmark.csproj [hunk 1]
```diff
@@ -3,16 +3,34 @@
...
+    <Configurations>Debug;Release;x86</Configurations>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x86'">
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
```
Comment (gfoidl) [inline Benchmark/Benchmark.csproj:16]: These can be simplified to just
```suggestion
  <PropertyGroup>
    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
  </PropertyGroup>
```
for all combinations.

So remove the other ones, just use the suggested one.

File: BenchmarkHandCoded/BenchmarkHandCoded.csproj [hunk 2]
```diff
@@ -3,13 +3,17 @@
...
     <DockerDefaultTargetOS>Windows</DockerDefaultTargetOS>
+    <Configurations>Debug;Release;x86</Configurations>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='x86|AnyCPU'">
```
Comment (gfoidl) [inline BenchmarkHandCoded/BenchmarkHandCoded.csproj:14]: Same here.

File: README.md [hunk 3]
```diff
@@ -50,35 +52,30 @@ AMD EPYC 7262, 1 CPU, 16 logical and 8 physical cores
...
+|        'FastFloat.Parse() - multicol' |  w-c-300K.csv | 493.40 ms | 5.625 ms | 4.697 ms | 487.38 ms |  0.84 |     1.23 |
 
 ```
 
 
 
 # Requirements
 
-.NET Core 3.1 or better. Under .NET 5 framework, the library takes advantage of the new Math.BigMul() function.
+.NET standard 2.0 or better. Under .NET 5 framework, the library takes advantage of the new Math.BigMul() & System.Runtime.Intrinsics.X86 SIMD functions.
```
Comment (gfoidl) [inline README.md:78]:
```suggestion
.NET standard 2.0 or newer. Under .NET 5 framework, the library takes advantage of the new Math.BigMul() & System.Runtime.Intrinsics.X86 SIMD functions.
```
Why "better"? I think "newer" is more appropriate.

File: TestcsFastFloat/TestcsFastFloat.csproj [hunk 4]
```diff
@@ -16,6 +19,10 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='x86|AnyCPU'">
```
Comment (gfoidl) [inline TestcsFastFloat/TestcsFastFloat.csproj:22]: See above.

**Review by gfoidl — COMMENTED** [review]
A hint for csproj-files.

File: BenchmarkHandCoded/BenchmarkHandCoded.csproj (hunk 2 above)
Comment (CarlVerret) [reply BenchmarkHandCoded/BenchmarkHandCoded.csproj]: btw thanks again for your interest in our projet !

**Pull request closed without merging**